vic/vic_run/src/snow_utility.c

```diff
@@ -208,18 +211,24 @@ new_snow_density(double air_temp)
     density_new = 0.0;
 
     if (options.SNOW_DENSITY == DENS_SNTHRM) {
+        // what are 67.9, 51.3, 2.6
         density_new = 67.9 + 51.3 * exp(air_temp / 2.6);
     }
     else if (options.SNOW_DENSITY == DENS_BRAS) {
-        air_temp = air_temp * 9. / 5. + 32.;
+        // why are we converting to F?
+        air_temp = C_TO_F(air_temp);
         if (air_temp > 0) {
+            // what?
             density_new = param.SNOW_NEW_SNOW_DENSITY + 1000. *
                           (air_temp / 100.) * (air_temp / 100.);
```

Comment:
@tbohn - any idea what is going on here?  I've gone through Bras pretty carefully and can't figure out what is going on here.